data/ecosystems/c/celo.toml

```diff
@@ -25,10 +25,112 @@ github_organizations = [
   "https://github.com/Ubeswap",
   "https://github.com/ReSource-Network",
   "https://github.com/Tradegen",
-  "https://github.com/centfinance"
+  "https://github.com/centfinance",
+  "https://github.com/NEFTME/",
+  "https://github.com/hyperlane-xyz/",
+  "https://github.com/celo-tracker",
 ]
 
-# Repositories
+# Respositories
+
+[[repo]]
+url = "https://github.com/mezumo-dca/dca-contracts"
+
+[[repo]]
+url = "https://github.com/NEFTME/neftme-poc-dapp"
+
+[[repo]]
+url = "https://github.com/hyperlane-xyz/abacus-monorepo"
```

Comment:
Hyperlane should be its own ecosystem. Added here: [e817889](https://github.com/electric-capital/crypto-ecosystems/commit/e8178891ae6d7c171a9dd0c7d16ac0870f7c2259)

Reply:
Hyperlane should just be its own top level ecosystem outside of Celo.  If there is a specific repo from Hyperlane in the Celo ecosystem, we could add it in the celo.toml file.